styles/styles.css

```diff
@@ -1535,15 +1555,52 @@ main .customer-journey-section .new-image-set > h5 {
 
   main .cards p:first-of-type {
     font-size: 14px;
+    -webkit-font-smoothing: antialiased;
   }
 
   main .section .default-content-wrapper h3 {
+    padding: 0 20px;
+  }
+
+  main .carve-career-all-section .default-content-wrapper h1 {
+    font-size: 24px;
+    padding: 0 20px;
+    text-align: center;
+  }
+
+  main .carve-career .default-content-wrapper h1 {
+    font-size: 24px;
+    padding: 0 20px;
+    text-align: center;
+  }
+
+  main .carve-career-all-section h3:nth-child(4) {
+    font-size: 15px;
     padding: 0;
   }
 
-  main .section .default-content-wrapper h1 {
+  main .carve-career-all-section .default-content-wrapper h3:nth-child(3) {
```

Comment:
preferably use semantics (if required different headings) to distinguish on the changes required rather than making them fixed via numbers